.github/workflows/pr-gpu.yaml

```diff
@@ -21,14 +21,14 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - name: "python3.11-pytorch2.5.1-gpus1"
+        - name: "python3.11-pytorch2.6.0-gpus1"
```

Comment:
please temp change `pull_request_target` to `pull_request` to show CI passing on the 2.5 images. then change back to `pull_request_target`